WordPress/gutenberg
# Mark FSE blocks as non experimental

## Description

Grab bag of plugin changes needed ahead of WP 5.9.

- Moves all of the FSE blocks used by Twenty Twenty-two from `__experimentalRegisterExperimentalCoreBlocks` to `registerCoreBlocks`.
- ~Removes `__experimentalRegisterExperimentalCoreBlocks` as there are no more experimental blocks.~
- ~Removes `GUTENBERG_PHASE` checks no longer needed for WP 5.9.~
- Fixes a bug in our webpack code that prefixes block PHP functions with `gutenberg_`. It did not work correctly when the function declaration is indented, causing duplicate function errors if you run the plugin against WP 5.9.

## How has this been tested?

1. Load the post editor. Check that blocks **including** FSE blocks appear.
2. Load the site editor. Check that blocks **including** FSE blocks appear.
3. Load the widgets editor. Check that blocks **excluding** FSE blocks appear.
4. Load the navigation editor. Check that blocks **excluding** FSE blocks appear.

## Checklist:
